### PR TITLE
CoreNFC Tag Reader

### PR DESCRIPTION
**Pull request by ivanherreragl**

### CODE REVIEW

**Comment by conceptdev** [comment]
Can we get a screenshot with an actual tag read?

File: ios11/NFCTagReader/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# NFCTagReader
```
Comment (conceptdev) [inline ios11/NFCTagReader/README.md]: use the other heading syntax (developer.xamarin.com/samples doesn't support the leadiing `#` but only for the first line)

```
NFCTagReader
==========
```

File: ios11/NFCTagReader/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+This sample shows how to integrate Core NFC Framework into your application to enable NFC tag reading.
+
+## Overview
+
+Core NFC provides NFC tag reading capabilities to your applications.  This sample application will read an NDEF formatted NFC Forum tag (Type 1 to 5) that contains an NDEF message.  Clicking on the **Scan** button will start the tag scanning process.  The application will list each NDEF message read from the NFC tag in the main message table view with a single row entry, and the row's label displays the number of NDEF record payloads in the message.
+
+Clicking on a row in the message table will display the payload table view with details on each of the NDEF record stored in the message.
+
+Here are the three main steps involved in enabling NFC tag reading using Core NFC:
```
Comment (conceptdev) [inline ios11/NFCTagReader/README.md]: three steps missing?

**Comment by conceptdev** [comment]
@ivanherreragl bump :) would love to have this published by the 19th

**Comment by ivanherreragl** [comment]
Should be ready to merge :)

**Pull request merged**